avahi-daemon/simple-protocol.c

```diff
@@ -264,7 +264,7 @@ static void dns_server_browser_callback(
 }
 
 static void handle_line(Client *c, const char *s) {
-    char cmd[64], arg[64];
+    char cmd[64], arg[256];
```

Comment:
I'm not sure what to do here. I still think it should be `AVAHI_DOMAIN_NAME_MAX` to make it consistent with the other API accepting domain names that aren't in the wire format. I guess it should be possible to bump it to 256 to address the most popular issue where long hostnames + '.local' can't fit but I'm not sure why the wire format limit should be used here.

Could you rebase it to trigger the CI? The simple protocol is tested by the smoke test with radamsa generating all sorts of commands with all sorts of arguments.